### ISSUE TITLE
Can't enable "Include issue numbers assigned to group"

### ISSUE THREAD

**Issue by tanertas** [akiko-pusu/redmine_issue_badge#96]
I want to get notified about all issues including groups that I'm being member of. But this checkbox in account settings is clickable but it remains unchecked.

**Comment by akiko-pusu** [comment]
Hi, @tanertas. Thanks for reporting issue.
Could you please tell me the version of Redmine and this plugin?

**Comment by tanertas** [comment]
Ah, sorry. Redmine version is 4.0.4. I noticed that this problem is not specifically related with enabling groups. When I globally disable redmine_issue_badge for users, users can't enable it from their account settings. Enabling it from personal settings doesn't have any affect.

**Comment by akiko-pusu** [comment]
I could not reproduce this situation.
Could you please tell me if you use any other plugins?

**Comment by tanertas** [comment]
Maybe related with db engine? I use PostgreSQL.
I use these plugins:
issue_closer, lightbox2, checklist, auto_assign_group, editauthor

**Comment by akiko-pusu** [comment]
Ok, thanks for the above information.
DB information is valuable for me.

**Comment by akiko-pusu** [comment]
@tanertas I could not reproduce the situation.
Are there any plugin that extend user profile like this issue badge plugin?

**Comment by tanertas** [comment]
No, only issue badge plugin has special field.

**Comment by akiko-pusu** [comment]
Thanks for above comment.
Which browser do you use? (Does this problem occur in any browser?)

**Comment by akiko-pusu** [comment]
FYI: Here is the DDL for issue_badge_user_settings table. (PostgreSQL)
Is there any difference? 

```
create table issue_badge_user_settings
(
  id                     serial not null
    constraint issue_badge_user_settings_pkey
    primary key,
  user_id                integer,
  enabled                boolean,
  created_on             timestamp,
  updated_on             timestamp,
  show_assigned_to_group boolean,
  badge_order            integer default 0
);

create index index_issue_badge_user_settings_on_user_id
  on issue_badge_user_settings (user_id);

```

**Comment by tanertas** [comment]
It looks correct:
```
redmine=# \d+ issue_badge_user_settings
                                                                  Table "public.issue_badge_user_settings"
         Column         |            Type             | Collation | Nullable |                        Default                        | Storage | Stats target | Description 
------------------------+-----------------------------+-----------+----------+-------------------------------------------------------+---------+--------------+-------------
 id                     | integer                     |           | not null | nextval('issue_badge_user_settings_id_seq'::regclass) | plain   |              | 
 user_id                | integer                     |           |          |                                                       | plain   |              | 
 enabled                | boolean                     |           |          |                                                       | plain   |              | 
 created_on             | timestamp without time zone |           |          |                                                       | plain   |              | 
 updated_on             | timestamp without time zone |           |          |                                                       | plain   |              | 
 show_assigned_to_group | boolean                     |           |          |                                                       | plain   |              | 
 badge_order            | integer                     |           |          | 0                                                     | plain   |              | 
Indexes:
    "issue_badge_user_settings_pkey" PRIMARY KEY, btree (id)
    "index_issue_badge_user_settings_on_user_id" btree (user_id)

```

**Comment by akiko-pusu** [comment]
Thanks for above info!

**Comment by akiko-pusu** [comment]
Related issue #97 
Could you try this patch if possible?
https://github.com/akiko-pusu/redmine_issue_badge/issues/97#issuecomment-523752443

**Comment by akiko-pusu** [comment]
Workaround is merged into master on commit: 59bd9f3.
Please try and confirm how it works.
